### ISSUE TITLE
Is there a way to get JSON output?

### ISSUE THREAD

**Issue by bowencool** [mwilliamson/mammoth.js#407]
Hi, I'm using the [Slate](https://github.com/ianstormtaylor/slate) editor, which expects a JSON input as its initial value. Therefore, I am converting an HTML string into JSON.

However, I think it would be better to get a JSON output directly from this project so that I don't need to compile the HTML again.

**Comment by mwilliamson** [comment]
Nope, there's no way to get JSON output: the HTML is always outputted as a string.
